source/mir/bignum/integer.d

```diff
@@ -317,6 +318,116 @@ struct BigInt(size_t maxSize64)
         return overflow;
     }
 
+    /++
+    Performs division & extracts the remainder.
+    Params:
+        rhs = unsigned value to divide by
+    Returns:
+        unsigned remainder value
+    +/ 
+    auto divRem(size_t rhsSize)(UInt!rhsSize rhs)
```

Comment:
This function should be removed for BigInt because it returns a very fat argument. Let `opOpAssign` call `divm` directly.